apis/Google.Storage.V1/Google.Storage.V1.IntegrationTests/UrlSignerTest.cs

```diff
@@ -54,7 +75,7 @@ private void DeleteTest_InitDelayTest()
             var name = GenerateName();
             string url = null;
 
-            _fixture.RegisterDelayTest(GetTestName(nameof(DeleteTest)), _duration,
+            _fixture.RegisterDelayTest(GetDelayTestName(), _duration,
```

Comment:
Good idea. Done. I did need to add some 'global' state to the test fixture so the register calls don't need to pass in the declaring class name though, but I don't think its too bad.